resources.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+
+	"gopkg.in/urfave/cli.v2"
+)
+
+var get getResponse
+
+type resourceParams struct {
+	Tags []string `json:"tags"`
+}
+
+func createResource(c *cli.Context, resourceType string) {
+	params := makeBody(c)
+	var table [][]string
+	switch resourceType {
+	case "Folders":
+		table = getFolders(params)
+	case "Sites":
+		table = getSites(params)
+	default: //"Browsers"
```

Comment:
Let "Browsers" be an actual case and let the default either panic or return an error.